pkg/models/kanban_task_bucket_test.go

```diff
@@ -157,3 +157,22 @@ func TestTaskBucket_Update(t *testing.T) {
 		}, false)
 	})
 }
+
+func TestTaskBucket_UpsertDuplicateError(t *testing.T) {
+	t.Run("test constraint violation detection", func(t *testing.T) {
```

Comment:
Init db fixtures, then insert a task that's already in one of the fixture buckets, then assert that the error is returned correctly